docs/cover.md

```diff
@@ -48,10 +48,10 @@ To install the MapboxNavigation framework using [CocoaPods](https://cocoapods.or
 1. Create a [Podfile](https://guides.cocoapods.org/syntax/podfile.html) with the following specification:
    ```ruby
    # Latest stable release
-   pod 'MapboxNavigation', '~> ${MINOR_VERSION}'
+   pod 'MapboxNavigation', '~> ${SHORT_VERSION}'
```

Comment:
> Cocoapods recognize 1.4 as a valid SEMVER, but it doesn't hurt to
> specify patch version (1.4.0) as well.

Apparently this causes CocoaPods to accept new patch versions but not new minor versions. CocoaPods.org recommends just specifying the minor version in its installation instructions, so consistency with that would lead to less confusion. Any downside to keeping this line the same but changing the others?